console/frontend/src/main/frontend/src/app/components/pages/pages-footer/pages-footer.component.html

```diff
@@ -1,6 +1,6 @@
 <div class="footer">
   <div>
-    <strong>Frank</strong><strong class="text-frank">!</strong><strong>Console</strong> powered by:
+    <strong>FF!</strong><strong>Console</strong> {{ consoleVersion }} | powered by:
```

Comment:
```suggestion
    <strong>FF!</strong>&nbsp;<strong>Console</strong> {{ consoleVersion }} | powered by:
```
@philipsens hoe ziet de naming conventie er uit?

Reply:
Jup FF! [spatie] Console